README.md

```diff
@@ -208,6 +208,12 @@ Allows custom monitoring events to be added into the Node Application Metrics ag
 * `type` (String) the name you wish to use for the data. A subsequent event of that type will be raised, allowing callbacks to be registered for it.
 * `data` (Object) the data to be made available with the event. The object must not contain circular references, and by convention should contain a `time` value representing the milliseconds when the event occurred.
 
+### appmetrics.writeHeapSnapshot([filename],[callback])
+Dumps the v8 heap via `heapdump`. For more information, see [the heapdump README](https://github.com/bnoordhuis/node-heapdump/blob/master/README.md)
+
+### appmetrics.triggerNodeReport()
```

Comment:
I think it would be better to export `appmetrics.nodereport = require('nodereport')`, so we have the full nodereport API available. The advantage of having these modules in appmetrics is that they are pre-compiled (which is huge), not that the API is wrapped. Particularly with nodereport which runtimes and the community are building together and designing the API. heapdump is a bit different, it only has one method, and it hasn't changed in forever.